svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -85,50 +68,26 @@ class LoadOutputAsLayerDialog(QDialog, FORM_CLASS):
     def __init__(self, iface, output_type=None, path=None, mode=None):
 
         # sanity check
-        if (output_type is not None
-                and output_type not in OQ_ALL_LOADABLE_TYPES):
+        if output_type not in OQ_ALL_LOADABLE_TYPES:
             raise NotImplementedError(output_type)
         self.iface = iface
         self.path = path
-        self.npz_file = None
+        # self.npz_file = None
         self.output_type = output_type
```

Comment:
Should this be commented or not? Usually one sets the attribute to None with a comment that the value is set in subclasses.